tests/lib/rules/no-irregular-whitespace.js

```diff
@@ -171,6 +171,28 @@ ruleTester.run("no-irregular-whitespace", rule, {
         { code: "const error = `\n\u3000\n`;", options: [{ skipTemplates: true }], parserOptions: { ecmaVersion: 6 } },
         { code: "const error = `foo\u3000bar\nfoo\u3000bar`;", options: [{ skipTemplates: true }], parserOptions: { ecmaVersion: 6 } },
 
+        { code: "<div>\u000B</div>;", options: [{ skipJSXText: true }], parserOptions: { ecmaFeatures: { jsx: true } } },
```

Comment:
Can you also add `invalid` tests to confirm that this would be reported when `skipJSXText` is not set to `true`?

```
{ code: "<div>\u000B</div>;", parserOptions: { ecmaFeatures: { jsx: true } }, errors: ... }
{ code: "<div>\u000B</div>;", options: [{}], parserOptions: { ecmaFeatures: { jsx: true } }, errors: ... }
{ code: "<div>\u000B</div>;", options: [{ skipJSXText: false }], parserOptions: { ecmaFeatures: { jsx: true } } errors: ... },
```

Reply:
Fixed